TestingExplainer/ViewController.swift

```diff
@@ -225,6 +227,7 @@ class OverlayViewController: UIViewController {
             highlightBorderView?.layer.borderWidth = 5
             self.view.addSubview(highlightBorderView!)
             
+                            // Last I remember (And google from 2012 agrees) bounds is not strictly KVO compliant  
```

Comment:
yeah i had trouble finding a definitive source of what is KVO compliant, but nothing in UIKit is guaranteed from what I understand, just some properties work. Looking through the docs maybe `layoutSublayers(of:)` would be a better trigger?